textord/tospace.cpp

```diff
@@ -932,6 +932,9 @@ ROW *Textord::make_prop_words(
   box_it.set_to_list (row->blob_list ());
   word_it.set_to_list (&words);
   bol = TRUE;
+  fuzzy_non = FALSE;
```

Comment:
Ray addressed that issue after Martin's  bug report: commit 97080412f fixes all three cases, so this pull request can be closed.